api/src/Tests/Traits/AppTestTrait.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+namespace App\Tests\Traits;
+
+use DI\Container;
+use InvalidArgumentException;
+use JsonException;
+use PHPUnit\Framework\MockObject\MockObject;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Message\UriInterface;
+use Slim\App;
+use Slim\Psr7\Factory\ServerRequestFactory;
+use UnexpectedValueException;
+
+use Slim\Http\Request;
+use Slim\Http\Response;
+use Slim\Http\Environment;
+
+require __DIR__.'/../../App/Routes.php';
+require __DIR__.'/../../App/Dependencies.php';
+
+require __DIR__.'/../../../../functions/functions.inc';
+
+class BlankMiddleWare
+{
+
+
+    /* BlankMiddleWare */
+
+    public function __invoke(Request $request, Response $response, $next)
+    {
+        return $next($request, $response);
+
+    }
+
+
+    /* End */
+}
+
+/**
+ * App Test Trait.
+ */
+trait AppTestTrait
```

Comment:
TIL!

(Rails would call this a Module, same basic behaviour. Beautiful).

Reply:
yay!